.github/workflows/diktat_snapshot.yml

```diff
@@ -56,9 +56,6 @@ jobs:
           mv gradle/libs.versions.toml gradle/libs.versions.toml_current
           mv build/diktat-snapshot/libs.versions.toml gradle/libs.versions.toml
 
-          mv gradle/plugins/src/main/kotlin/com/saveourtool/diktat/buildutils/diktat-convention-configuration.gradle.kts gradle/plugins/src/main/kotlin/com/saveourtool/diktat/buildutils/diktat-convention-configuration.gradle.kts_current
```

Comment:
it was just added, why remove now?

Reply:
Snapshot was built from master with new id for gradle plugin. So no need to override it now